json-rpc/src/methods.rs

```diff
@@ -388,6 +393,20 @@ impl<'a> Handler<'a> {
         data::get_network_status(self.service.role.as_str())
     }
 
+    /// Returns all resources in the account specified by `params`
+    async fn get_resources(
+        &self,
+        params: GetResourcesParams,
+    ) -> Result<BTreeMap<String, AnnotatedMoveStruct>, JsonRpcError> {
```

Comment:
I suppose this is intended to just return an empty map in the case where the account doesn't exist?

Reply:
Correct. We could return an error instead, but there's no concept of an account "existing" without containing some resources, so empty seems reasonable?

Reply:
we don't return error for get_account when the address is not an account address, so return empty is consistent with existing api